api/src/routes/anime/animeRoutes.ts

```diff
@@ -0,0 +1,15 @@
+import express from "express";
+import { animeController } from "@@root/controllers";
+import { extractPaginationParams } from "@@root/middleware";
+
+const animeRoutes = express.Router();
+
+animeRoutes.route("/anime").get(extractPaginationParams, animeController.index);
+
+animeRoutes.route("/anime/:id")
```

Comment:
Can you change this to `/anime/id/:id`? 

After using the Discord API for images, they used a routing structure of displaying the name of the field followed by the value. So if a route needed a channel ID and a guild ID it would look like `/someEndpoint/channel_id/<channel id value>/guild_id/<guild id value>`. For me this helped visually separate what value needed to be where before even heading to the documentation